.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Jira Ticket: http://jira.nypl.org/browse/RENO-XXX
+
+## **This PR does the following:**
+-
```

Comment:
I'm not sure we need it, because we'll have the link to the figma design, and a link to view the component live on a multidev, so any screenshot we add could become quickly outdated, and then cause more confusion. We don't have the multidevs setup yet, but I think once we do, this will make more sense to you.

Reply:
No, fair enough, when I commented this we hadn't had retro yet and I'd forgotten about that being the end goal here.